cache_cleaner/utils.py

```diff
@@ -0,0 +1,40 @@
+import errno
+import os
+import re
+
+
+def parse_max_cache_size_arg(value):
+    m = re.match(r"([0-9]+)([KMGT])", value)
+    if m is not None:
```

Comment:
nit: this needs an `else:` branch throwing a ValueError if it doesn't work.

Reply:
I've just realised this isn't actually the case - it wasn't my code originally, and as it was it was able to support byte values with the suffices being optional helpers. tldr it's not necessarily a value error if the suffix isn't there.